pipelines/_ingredients/ApVerifyWithFakes.yaml

```diff
@@ -5,7 +5,6 @@ description: Fully instrumented AP pipeline with fakes
 imports:
   - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipeWithFakes.yaml
   # Most metrics should not be run with fakes, to avoid bias or contamination.
-  - location: $AP_VERIFY_DIR/pipelines/_ingredients/MetricsForFakes.yaml
```

Comment:
Please rearrange the commits so that this import is removed *before* `MetricsForFakes.yaml`. Otherwise, the pipeline is in a broken state.

Reply:
It might be worth mentioning in [the commit message body](https://developer.lsst.io/work/flow.html#writing-commit-message-body-content) that the new metrics come from `ApPipeWithFakes`, not `ap_verify`. That confused me until I could trace the imports.